BoxContentSDK/BoxContentSDK/Operations/BOXStreamOperation.m

```diff
@@ -148,7 +148,9 @@ - (long long)contentLength
 
 - (void)abortWithError:(NSError *)error
 {
+    self.error = error;
     [self.sessionTask cancel];
+    //FIXME: Remove?
```

Comment:
We spoke offline. Basically, canceling the session task will call the didFinish:... (see documentation).